### DISCUSSION TITLE
Making "things" distro agnostic

### DISCUSSION THREAD

**Discussion by nandgator** [elementary/triage#64] [Ideas]
Thanks to this amazing team and community, elementary OS has catapulted. The fact that Linux has so many distribution is the proof of "power of open source & creativity". But as y'all know diversity gives rise to inter-compatibility issue. This is being tackled by projects like Flatpak, AppImages and Snaps and seeing elementary move towards Flatpak is really cool!

My idea is to package all the software paraphernalia into distributable form - one for users and other for developers. Here are two _things_ that come to my mind:

- The Pantheon DE - Build docker images for quick development and testing (this might take some serious time as there _might_ be a shift to Wayland & GTK4 beforehand), then make it distro agnostic.
- elementary Apps - These are going Flatpak and that's good for any user, but here also a developer might prefer docker images with build tools.

Tell me what do you think?

**Comment by danirabbit** [comment]
Hey thanks for your suggestion! We’re currently tracking issues related to distro agnosticism in this project: https://github.com/orgs/elementary/projects/116

since this is already something that is actively worked on, I’m gonna close this as resolved :)
